src/io/config_auto.cpp

```diff
@@ -763,6 +764,7 @@ const std::string Config::DumpAliases() {
   str_buf << "\"task\": [\"task_type\"], ";
   str_buf << "\"objective\": [\"objective_type\", \"app\", \"application\", \"loss\"], ";
   str_buf << "\"boosting\": [\"boosting_type\", \"boost\"], ";
+  str_buf << "\"data_sample_strategy\": [], ";
```

Comment:
I'm surprised to see that this is empty, given that `config.h` has the following:

```cpp
// options = bagging, goss
```

Could you please try re-running the parameter generator?

```python
python helpers/parameter_generator.py
```

Reply:
@jameslamb Everything is OK here. These are aliases of the parameter main name. And I can't see any aliases for the new `data_sample_strategy` parameter:

![image](https://user-images.githubusercontent.com/25141164/148568595-f001dc6d-00fa-467a-89e3-d6b79334b398.png)

Compare with `boosting` parameter, for example:

![image](https://user-images.githubusercontent.com/25141164/148568730-f1378cde-c123-4d7e-9848-2e51f94fe101.png)

Reply:
ooooooo I was mistaking them for the "options". Ok thank you! Sorry about that.